doc/source/whatsnew/v0.19.0.txt

```diff
@@ -323,6 +323,22 @@ Other enhancements
                        index=['row1', 'row2'])
      df.sort_values(by='row2', axis=1)
 
+- ``Index`` now supports ``.dropna`` to exclude missing values (:issue:`6194`)
```

Comment:
make a sub section I think (and maybe roll some other API changes there from Index eg where) might be others